odoo/addons/base/models/res_partner.py

```diff
@@ -986,7 +986,7 @@ def open_commercial_entity(self):
     )
     def _compute_display_name(self):
         for partner in self:
-            name = partner.name
+            name = partner.name or ''
```

Comment:
NIt: since you have to touch line (maybe move the line after `if partner._context.get("formatted_display_name"):` which makes the code structure in `if` and `else ` the same?

Reply:
make sense, I've updated it